docs/extend/plugins_volume.md

```diff
@@ -131,47 +151,75 @@ Respond with a string error if an error occurred.
 ```
 
 Docker requires the plugin to provide a volume, given a user specified volume
-name. This is called once per container start. If the same volume_name is requested
+name. `Mount` is called once per container start. If the same `volume_name` is requested
 more than once, the plugin may need to keep track of each new mount request and provision
 at the first mount request and deprovision at the last corresponding unmount request.
 
 `ID` is a unique ID for the caller that is requesting the mount.
 
 **Response**:
-```json
-{
-    "Mountpoint": "/path/to/directory/on/host",
-    "Err": ""
-}
-```
 
-Respond with the path on the host filesystem where the volume has been made
-available, and/or a string error if an error occurred.
+- **v1**:
+
+  ```json
+  {
+      "Mountpoint": "/path/to/directory/on/host",
+      "Err": ""
+  }
+  ```
+
+- **v2**:
+
+  ```json
+  {
+      "Mountpoint": "/path/under/PropagateMount",
+      "Err": ""
+  }
+  ```
+
+`Mountpoint` is the path on the host (v1) or inside the plugin (v2) where the
```

Comment:
Better to rephrase as:
`Mountpoint` is the path on the host (v1) or path in the plugin that has been propagated to the host (v2), where the volume has been made available.`